Build/Chakra.Build.Paths.props

```diff
@@ -8,6 +8,12 @@
     <BuildConfigPropsPath>$(MSBuildThisFileDirectory)</BuildConfigPropsPath>
 
     <ChakraBuildPathImported>true</ChakraBuildPathImported>
-
+  </PropertyGroup>
+  <PropertyGroup Label="ICU">
```

Comment:
Possibly, in theory these options control where to look for ICU on disk, and could be used even if Intl is disabled. I'll look into the various options.